lib/ruby-debug-ide/attach/util.rb

```diff
@@ -83,15 +83,19 @@ def filter_ruby_processes(pids)
     ruby_processes.add(pid.to_i)
   end
 
+  non_ruby_processes = Array.new
+
   pids.each do |pid|
```

Comment:
let's use the functional style here since we're iterating anyway.
I think http://ruby-doc.org/core-2.3.0/Enumerable.html#method-i-partition suits the needs perfectly.